AppCenter/AppCenter/Internals/Util/MSUtility+StringFormatting.m

```diff
@@ -12,8 +12,20 @@
  */
 NSString *MSUtilityStringFormattingCategory;
 
+// We support the following formats:
+// target=<..>
+// appsecret=<..>
+// target=<..>;appsecret=<..>
+// ios=<..>;macos=<..>
+// targetIos=<..>;targetMacos=<..>
+// ios=<..>;macos=<..>;targetIos=<..>;targetMacos=<..>
+
 static NSString *kMSTransmissionTargetKey = @"target=";
 static NSString *kMSAppSecretKey = @"appsecret=";
+static NSString *kMSAppSecretIosKey = @"ios=";
+static NSString *kMSAppSecretMacosKey = @"macos=";
+static NSString *kMSTransmissionIosTargetKey = @"targetIos=";
+static NSString *kMSTransmissionMacosTargetKey = @"targetMacos=";
```

Comment:
Let's not worry about the default target for Catalyst at this point. It will simplify the logic. I updated the WI.

Reply:
I think I'm going to leave this one to @annakocheshkova (hope I didn't break logic with https://github.com/microsoft/appcenter-sdk-apple/pull/2073/commits/45fa8ead00e24c806cc4e3308d1dced86f98c595 😃)